src/resources/2017/AsdonMartin.ts

```diff
@@ -182,6 +195,41 @@ export function fillTo(targetUnits: number): boolean {
   return getFuel() >= targetUnits;
 }
 
+function fillWithBestInventoryItem(targetUnits: number): boolean {
+  const options = inventoryItems().sort(
+    (a, b) =>
+      getAverageAdventures(b) / autosellPrice(b) -
+      getAverageAdventures(a) / autosellPrice(a)
+  );
+  if (options.length === 0) return false;
+
+  const best = options[0];
+  if (autosellPrice(best) / getAverageAdventures(best) > 100) return false;
+
+  const amountToUse = clamp(
+    Math.ceil(targetUnits / getAverageAdventures(best)),
+    0,
+    itemAmount(best)
+  );
+  return insertFuel(best, amountToUse);
+}
+
+/**
+ * Fill your Asdon Martin by prioritizing mallmin items in your inventory. Default to the behavior of fillTo.
+ * @param targetUnits Fuel level to attempt to reach.
+ * @returns Whether we succeeded at filling to the target fuel level.
+ */
+export function fillWithInventory(targetUnits: number): boolean {
```

Comment:
The other fn ends with `to` so either rename this to `fillWithInventoryTo` or rename the other one.